src/System.Text.Json/tests/Utf8JsonWriterTests.cs

```diff
@@ -6777,28 +6882,29 @@ private static string GetGuidsExpectedString(bool prettyPrint, string keyString,
             return Encoding.UTF8.GetString(ms.ToArray());
         }
 
-        private static string GetNumbersExpectedString(bool prettyPrint, int numberOfElements)
+        private static string GetNumbersExpectedString<T>(bool prettyPrint, int numberOfElements, T value)
         {
-            var ms = new MemoryStream();
-            TextWriter streamWriter = new StreamWriter(ms, new UTF8Encoding(false), 1024, true);
+            var sb = new StringBuilder();
+            StringWriter stringWriter = new StringWriter(sb);
 
-            var json = new JsonTextWriter(streamWriter)
+            var json = new JsonTextWriter(stringWriter)
             {
                 Formatting = prettyPrint ? Formatting.Indented : Formatting.None,
             };
 
             json.WriteStartArray();
             for (int i = 0; i < numberOfElements; i++)
             {
-                json.WriteValue(1234567);
+                json.WriteValue(value);
             }
             json.WriteEnd();
 
             json.Flush();
 
-            return Encoding.UTF8.GetString(ms.ToArray());
+            return sb.ToString();
         }
 
+
```

Comment:
nit: remove extra new line

Reply:
Removed.

> Running CI legs...

I don't have access. Restricted only to Microsoft folks :-) ... hence was trying to push my luck with dotnet-bot